client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -270,6 +270,59 @@ function MetadataListEditingDirective() {
     };
 }
 
+MetadataSliderDirective.$inject = ['desks'];
+function MetadataSliderDirective(desks) {
+    return {
+        scope: {
+            list: '=',
+            disabled: '=ngDisabled',
+            item: '=',
+            field: '@',
+            change: '&'
+        },
+        templateUrl: 'scripts/superdesk-authoring/metadata/views/metadata-slider.html',
+        link: function(scope) {
+            desks.initialize().then(function() {
```

Comment:
I had a problem, when I open item it breaks, and console display's 'undefined item' error, but when I reload page it works fine... So I call promise to wait for desk initialize and then run function. Please, if you have some idea for solving problem in another way, fire it up!